pkg/v1/mutate/mutate.go

```diff
@@ -437,6 +437,13 @@ func layerTime(layer v1.Layer, t time.Time) (v1.Layer, error) {
 		}
 
 		header.ModTime = t
+
+		//PAX and GNU Format support additional timestamps in the header
+		if header.Format == tar.FormatPAX || header.Format == tar.FormatGNU {
```

Comment:
Is it worth always setting these times, even if it's not a PAX tar?

Reply:
Since they only exist in those Formats, I don't know how the archive/tar package will react when trying to write a header for another while those fields are set. I would expect it to handle that gracefully, but since I'm not sure of that I programmed defensively